README.md

```diff
@@ -1,15 +1,23 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 # HelloMicroverse
 
-The main goals of this project are:
-The main goals of this project to understand these things: 
--How to create branches with different base branches.
-1-->To deal with version control.
-2-->Follow GitHub flow.
-3-->Open pull request.
-4-->Write descriptive to understand commit messages.
-5-->Write short, detailed and easy to understand descriptions of pull requests.
-6-->Demonstrate an ability to submit a project for code review.
+
+>The main goals of this project are:
+ - How to create branches with different base branches.
+ - To deal with version control.
+ - Follow GitHub flow.
+ - Open pull request.
+ - Write descriptive to understand commit messages.
+ - Write short, detailed and easy to understand descriptions of pull requests.
+ - Demonstrate an ability to submit a project for code review.
+# Getting started section
+  - In this project,3 files are added,
+
+  - A .html file that includes a header with the text "Hello Microverse!".
+
+  - A .css file that includes one class with styles for your header.
+
+  - A test.md file with the text "This file should be ignored by git".
```

Comment:
- [ ] Good implementation of changes required, but you missed some point in the `Getting started section`. In this section, you explain how the users can set up your project locally and what users need in their computer  (for example they want a text editor, git installed,...) 🙃 🙃

Reply:
okay thanks but i lost my all 3 review request